system-addon/content-src/components/Card/Card.jsx

```diff
@@ -31,7 +31,10 @@ class Card extends React.Component {
   }
   onLinkClick(event) {
     event.preventDefault();
-    this.props.dispatch(ac.SendToMain({type: at.OPEN_LINK, data: this.props.link}));
+    this.props.dispatch(ac.SendToMain({
+      type: at.OPEN_LINK,
+      data: Object.assign(this.props.link, {where: (event.ctrlKey || event.metaKey || event.button === 1) ? "tab" : "current"})
```

Comment:
Yeah, I didn't find one :(. We have support for "window" though via the context menu (not shortcut) (incl. private windows).

I think this is better than not having it for 56, but I will leave it up to you guys. I am fine either way and can dig in more.